geode-core/src/distributedTest/java/org/apache/geode/security/query/QuerySecurityUnauthorizedUserBindParameterDUnitTest.java

```diff
@@ -42,17 +43,16 @@ public void configureCache() {
   @Test
   public void userWithoutRegionAccessAndPassingInWrappedBindParameterShouldThrowException() {
     String query = "select v from $1 r, r.values() v";
-    String regexForExpectedException = ".*DATA:READ.*";
+    String regexForExpectedException = ".*values.*";
```

Comment:
why did this change from DATA:READ?  It looks like it's because we catch the returned exception, but I don't see where .*values.* is matched?

Reply:
The test is trying to (implicitly) invoke the method `values()` on the `Region` object within the query: `"select v from $1 r, r.values() v"`, that's where it comes from.